tests/themes/tests.lua

```diff
@@ -0,0 +1,18 @@
+--- Simple script to load/test a theme.
+-- This is used from tests/themes/run.sh.
+
+local awful = require("awful")
+
+local steps = {
+    function(count)
+        if count <= 2 then
+            awful.spawn("xterm")
+        elseif #client.get() == 2 then
+            return true
```

Comment:
Errors will cause `tests/run.sh` to fail already (and we have `set -e` here).
With https://github.com/awesomeWM/awesome/pull/1973 also warnings (e.g. deprecations) will cause a failure.